src/throttler.guard.ts

```diff
@@ -141,6 +141,11 @@ export class ThrottlerGuard implements CanActivate {
     ttl: number,
   ): Promise<boolean> {
     const { req, res } = context.getArgByIndex(2);
+    // add in escape route for GQL Fastify
+    // and for if res is not added to the context.
```

Comment:
Comments should always contain an uppercase as a starting letter and a dot to end the sentence with.

Besides that, I think you can ditch `for` here. `and for if res [...]`